stablehlo/dialect/StablehloOps.td

```diff
@@ -3073,16 +3073,16 @@ def StableHLO_UniformDequantizeOp : StableHLO_UnaryElementwiseOp<"uniform_dequan
       [InferTensorType, Pure], HLO_QuantizedIntTensor, TensorOf<[F32, BF16]>> {
```

Comment:
`TensorOf<[F32, BF16]>` doesn't look right. Same for `UniformQuantizeOp` above.

Reply:
That is an oversight. Used `HLO_Float`  instead to comply with the current spec. PTAL.